recipe/meta.yaml

```
@@ -8,59 +8,90 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/greykite-{{ version }}.tar.gz
   sha256: 27c74119900b186361cb4ccdd040eaf9383bc41f7b3781706600891df2c7c6e3
+  patches:
+    0001-setup.patch
```

could you please more the patch file in the `patches` folder?

```suggestion
    patches/0001-setup.patch
```